specs/agents/logging.md

```diff
@@ -0,0 +1,36 @@
+## Agent logging
+
+### `log_level` configuration
+
+Sets the logging level for the agent.
+
+This option is case-insensitive.
+
+|                |   |
+|----------------|---|
+| Valid options  | `trace`, `debug`, `info`, `warning`, `error`, `critical`, `off` |
```

Comment:
Was there any considerations for `warn` vs `warning`? Personally i prefer having it as `warn` and kind of in line with the method from browser's console object. https://developer.mozilla.org/en-US/docs/Web/API/Console/warn

Reply:
Yes, there was a conversation. It seems that most languages either use `warning`/`critical` or `warn`/`fatal`. The decision to go with `warning`/`critical` was fairly arbitrary but seems to cover most of the agents well. In fact, in the PHP agent, the default is `warn` but Sergey specifically recommended `warning` because the PHP agent accepts any shortening as long as the prefix is unambiguous.

Reply:
The PHP also uses warning/critical: https://www.elastic.co/guide/en/apm/agent/php/current/configuration.html#configure-logging